### PR TITLE
cfu: Provide a script to convert .inf entries to a .quirk block

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [X] Documentation

### CODE REVIEW

File: plugins/cfu/inf-to-quirk.py
Comment (superm1) [inline plugins/cfu/inf-to-quirk.py:1]: Maybe this should be in `contrib/`?

File: plugins/cfu/inf-to-quirk.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            if line.find(inf_key) != -1:
+                data[quirk_key] = line.split(",")[4].strip()
+    if is_cfu:
+        if comment:
+            print(f"# {comment}")
+        print(f"[{instance_id}]")
+        for key, value in data.items():
+            print(f"{key} = {value}")
+    else:
+        print("this is not a CFU firmware.inf")
```
Comment (superm1) [inline plugins/cfu/inf-to-quirk.py:52]:
```suggestion
        sys.exit("this is not a CFU firmware.inf")
```

**Review by superm1 — DISMISSED** [review]

File: plugins/cfu/inf-to-quirk.py
Comment (hughsie) [reply plugins/cfu/inf-to-quirk.py:1]: maybe renaming it contrib/cfu-inf-to-quirk.py?

Comment (superm1) [reply plugins/cfu/inf-to-quirk.py:1]: Sure just update readme for new location and name

**Pull request merged**